commune_app/all_models/chores.py

```diff
@@ -10,6 +11,7 @@ class Chore(models.Model):
     assign_to = models.ForeignKey(User, on_delete=models.CASCADE)
     passed = models.BooleanField()
     completed = models.BooleanField()
+    Commune = models.ForeignKey(Commune, on_delete=models.CASCADE, null=True, blank=True)
```

Comment:
in python, you can easily overwrite everything and in this case, you did the import of Commune in line # 3
and in this line you overwrite it.
I think it will be better to do it like:
```suggestion
    commune_id = models.ForeignKey(Commune, on_delete=models.CASCADE, null=True, blank=True)
```